src/mat.c

```diff
@@ -1276,6 +1297,7 @@ Mat_VarDelete(mat_t *mat, const char *name)
                 Mat_Critical("Cannot remove file \"%s\".",tmp_name);
             }
         }
+        free(tmp_name);
```

Comment:
In one of my use-cases I "misuse" Mat_Critical to never return to the caller (using setjmp/longjmp). In that case, the freeing is too late here.

Reply:
Changed to have the caller stack-allocate buffers and pass them in, to avoid having to bother with free().